pup-0002.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Docstring Style Conventions
+author: Austin Macdonald
+created: 28-Mar-2017
+status: proposed
+---
+
+## Summary
+
+This PUP documents docstring style conventions for Pulp 3.0.
```

Comment:
We could probably add a little more here, e.g. "The PUP documents the usage of Google-style docstrings for Pulp 3+" (3+ rather than 3.0 to be clear that this doesn't apply for only 3.0, and not 3.1 or later)